CMakeLists.txt

```diff
@@ -5,6 +5,8 @@ if(NOT CMAKE_BUILD_TYPE AND NOT DEFINED ENV{CMAKE_BUILD_TYPE})
   set(CMAKE_BUILD_TYPE "Debug" CACHE STRING "default build type")
 endif()
 
+set(CODESIGN_IDENTITY "" CACHE STRING "Sets the macOS code signing identity.  If set to something besides the empty string "", then the dynamic libraries put into the hog files will be signed using this identity.")
```

Comment:
this does not work because of the double quotes inside of the documentation string `""`, CMake does not like that

Reply:
Thanks for the help there!